logfire/_internal/config.py

```diff
@@ -258,6 +261,44 @@ class CodeSource:
     """
 
 
+@dataclass(init=False)
+class VariablesOptions:
```

Comment:
having both VariablesOptions and VariablesConfig is worrying

Reply:
I agree, I'm assuming you understand the difference between them, do you have a suggestion for alternative naming?